### DISCUSSION TITLE
HD300 Firmware

### DISCUSSION THREAD

**Discussion by seansd** [OpenSageTV/Community#128] [SageTV Extenders (HD300,HD200)]
One of my HD300's needs a configuration reset but it doesn't seem like it can connect to my server or to the internet to download the firmware. Does someone have a link to the latest HD300 firmware so I can just download it onto a USB drive?

See ...https://github.com/OpenSageTV/Community/discussions/208

**Comment by seansd** [comment]
Awesome thank you.  Do we have no common place for file sharing anymore?

**Reply by inouridr** [reply]
Did someone share a link?   You say thank you, but I don't see a comment from anyone else.
I need the HD300 Firmware as well.  Can you share where you found it?

Thanks!

**Reply by seansd** [reply]
I don't recall exactly but pretty sure someone directly messaged me.  I do have it locally.   I'll try to find somewhere to post it if at all possible a bit later.

**Reply by jusjoken** [reply]
@seansd 

If someone has these files and can email them to me I will upload to the opensagetv repo and post links in a thread here

**Reply by jusjoken** [reply]
@inouridr  @seansd 

I uploaded the ones I had and updated the first post with the link.  If there are others please let me know.

Ken

**Reply by seansd** [reply]
Ah perfect thank you, yeah I had the last "beta" one, which I think at this point we probably don't need to call "beta" ;-).

**Comment by inouridr** [comment]
Thanks a bunch, VERY helpful and MUCH appreciated!
